src/dmd/tokens.d

```diff
@@ -302,7 +303,7 @@ enum TOK : int
     vector,
     pound,
 
-    // 239
+    // 241
```

Comment:
You only added one token, why does the count go up by two?

Reply:
IIRC `fileFullPath` had been added without upading these numbers in the past, so i also correct that; however I'd super like to just get rid of these numbers, they serve no purpose and get out of sync easily

Reply:
Fair enough, just making sure you hadn't typo'd. I think they have some use when trying to look up what Tok you have in some function but I'll leave it up to other to determine what to do.

Reply:
I see; in this case one could use instead arbitrary increasing numbers at sparse intervals, it's much more robust (and more stable across different compiler versions)
```
enum TOK : int
{
    reserved = 0,
    // we can reset numbers arbitrarily
    leftParentheses = 100,
    rightParentheses, // will be 101count)
    leftBracket = 200,
}
```